app/components/collections/edit_license_component.html.erb

```diff
@@ -38,4 +38,71 @@
     <p class="terms-of-use">
       <%= Settings.access.use_and_reproduction_statement %>
     </p>
+
+    <hr>
+
+    <p>
+      The option below will allow you either to:
+    </p>
+
+    <ol>
+      <li>Provide a custom use statement that will appear on every deposit in this collection or</li>
+      <li>allow depositors to enter a custom use statement, typically one required by a publisher in order to deposit an Open Access version of an article.</li>
+    </ol>
+
+    <p>
+      You will also be able to provide custom instructions for this field if you wish.
+    </p>
+
+    <fieldset>
+      <legend class="fs-6"><strong>Include custom use statement?</strong></legend>
+
+      <div class="ms-2">
+        <div class="form-check<%= " is-invalid" if error? %>">
+          <%= form.radio_button :allow_custom_rights_statement, false, class: "form-check-input", data: {action: "custom-rights#updateChooseCustomRightsFieldsetEnabled", custom_rights_target: "noIncludeCustomRightsRadioButton"} %>
+          <%= form.label :allow_custom_rights_statement_false, "No", class: "form-check-label" %>
+        </div>
+        <div class="form-check<%= " is-invalid" if error? %>">
+          <%= form.radio_button :allow_custom_rights_statement, true, class: "form-check-input", data: {action: "custom-rights#updateChooseCustomRightsFieldsetEnabled", custom_rights_target: "yesIncludeCustomRightsRadioButton"} %>
+          <%= form.label :allow_custom_rights_statement_true, "Yes", class: "form-check-label" %>
+        </div>
+
+        <div class="ms-4">
+          <fieldset data-custom-rights-target="customRightsConfigFieldset">
+            <legend class="fs-6"><strong>Choose the source of the custom use statement</strong></legend>
+            <div class="form-check<%= " is-invalid" if error? %>">
+              <%= form.radio_button :custom_rights_statement_source_option, "provided_by_collection", checked: (custom_rights_statement_source_option == "provided_by_collection"), class: "form-check-input", data: {action: "custom-rights#updateCustomRightsInstructionsFieldsetEnabled", custom_rights_target: "customRightsSourceIsCollectionButton"} %>
+              <%= form.label :custom_rights_statement_source_option_provided_by_collection, "Provide specific terms", class: "form-check-label pe-2",
```

Comment:
```suggestion
              <%= form.label :custom_rights_statement_source_option_provided_by_collection, "Provide specific terms", class: "form-check-label pe-2",  style: "vertical-align: top",
```

Reply:
This ensures the label is aligned next to the radio button (else it will float down because the text box takes more than one line).